ibis/tests/all/test_client.py

```diff
@@ -38,3 +39,20 @@ def test_query_schema(backend, con, alltypes, expr_fn, expected):
     expected = ibis.schema([(name, dtype(nullable=schema[name].nullable))
                             for name, dtype in expected])
     assert query.schema().equals(expected)
+
+
+@pytest.mark.parametrize('field', [
+    'bool_col', 'tinyint_col', 'smallint_col',
+    'int_col', 'bigint_col', 'float_col', 'double_col',
+    'date_string_col', 'string_col', 'timestamp_col'
+])
+@tu.skipif_unsupported
+def test_sql(backend, con, alltypes, field):
+    try:
+        result = con.sql(alltypes.compile())
+        getattr(result, field)
+        getattr(alltypes, field)
+    except AttributeError:
+        # the method couldn't be set as unsupported but maybe it is not
+        # implemented so it will raise AttributeError.
+        ...
```

Comment:
This should be `pytest.skip(reason)`, so that the test doesn't appear to pass for everything.

Reply:
makes sense, thanks. I will do ti!